src/components/main/HourlySection.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useMemo, useState } from 'react';
+import { convertTime } from 'utils/helpers';
+import styles from './Main.css';
+import MainHourlyCard from './MainHourlyCard';
+import { HourlyWeatherType } from 'types/weatherTypes';
+
+type HourlySectionProps = {
+  weather: HourlyWeatherType[];
+};
+
+const getHourLabels = (weather: HourlyWeatherType[]): string[] => {
+  const labels: string[] = [];
+
+  weather.forEach((el: HourlyWeatherType, i: number) => {
+    if (i === 0 || i === 7 || i === 15 || i === 23) {
```

Comment:
i'd recommend to use hours value for this condition (in case api will return disordered array), looks good for now. also you can use here array methods chaining to make this function look shorter and more declarative way

Reply:
But I don't now smth about hours. Hours can start from 10 or 1 or 23. This condition adapted for any variant. 
You are right. If I get disordered array, I will have a problem because in UI I'll have values like '17.00 11.00 01.00 15.00'
In this case I only need to add sort method by hours before start function `getHourLabels`